### DISCUSSION TITLE
How to have dokku route to a locally running container?

### DISCUSSION THREAD

**Discussion by IgnisDa** [dokku/dokku#5609] [Q&A] [answered]
I have to deploy a project that I am unable to deploy using dokku apps (due to various errors). However it does provide a `docker-compose.yml` that can be used to start up containers. I was wondering if I can use just the virtual apps feature of dokku to route to that running container. So `https://smith.mydomain.com` should route to the container running on port `8081` on my server.

Is the above possible?

**Comment by josegonzalez** [comment, marked as answer]
You can use [this method](https://dokku.com/docs/networking/network/#routing-an-app-to-a-known-ipport-combination) to route to a known ip/port combination.
